### ISSUE TITLE
Vesicoureteral reflux grade IV

### ISSUE THREAD

**Issue by RichardKlaus** [obophenotype/human-phenotype-ontology#6828]
**Preferred term label:**
Vesicoureteral reflux grade IV

**Synonyms**
VUR grad IV, VUR IV

**Definition (free text, please give PubMed ID)**
Vesicoureteral reflux with more-extensive dilation with blunting of the calyces and tortuosity of the ureter.
PMID: 3975102, 18322164

**Parent term (use hpo.jax.org/app)**
High-grade vesicoureteral reflux (newly proposed term)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-2234-4248

**Comment by pnrobinson** [comment]
added
